cmake/f3dConfig.cmake.in

```diff
@@ -13,13 +13,15 @@
 #   f3d_MODULE_RAYTRACING          Will be enabled if F3D was built with raytracing support
 #   f3d_BINDINGS_PYTHON            Will be enabled if F3D was built with python bindings
 #   f3d_BINDINGS_JAVA              Will be enabled if F3D was built with java bindings
+#   f3d_CONFIG_DIR                 Path to F3D configuration directory, can be absolute or relative
 
 @PACKAGE_INIT@
 
 set(f3d_MODULE_EXTERNAL_RENDERING "@F3D_MODULE_EXTERNAL_RENDERING@")
 set(f3d_MODULE_RAYTRACING "@F3D_MODULE_RAYTRACING@")
 set(f3d_BINDINGS_PYTHON "@F3D_BINDINGS_PYTHON@")
 set(f3d_BINDINGS_JAVA "@F3D_BINDINGS_JAVA@")
+set(f3d_CONFIG_DIR "@f3d_config_dir@")
```

Comment:
all other variables are uppercase, shouldn't we have `F3D_CONFIG_DIR` instead?

Reply:
F3D_CONFIG_DIR is not a cmake option of F3D, that is what it is not upper case.

Reply:
ok